core/domain/suggestion_services_test.py

```diff
@@ -763,6 +763,41 @@ def test_check_can_resubmit_suggestion(self):
             self.suggestion_id, self.normal_user_id)
         self.assertEqual(can_resubmit, False)
 
+    def test_update_translation_updates_translation_suggestion_change(self):
```

Comment:
The method under test is `update_translation_suggestion`, so the first part of the test name should reflect that.

Reply:
Done